docs/dunst.5.pod

```diff
@@ -364,7 +364,7 @@ Hide the count of stacked duplicate notifications.
 Show an indicator if a notification contains actions and/or open-able URLs. See
 ACTIONS below for further details.
 
-=item B<icon_position> (values: [left/right/off], default: off)
+=item B<icon_position> (values: [left/right/top/off], default: off)
```

Comment:
It seems like this documentation was wrong. The default icon position was "left" before. Could you update this and set the default icon position to left in `notification.c` (`notification_create()`).